spec/language-file.md

```diff
@@ -98,25 +101,62 @@ An optional object containing two properties:
 
 ### metadata.sources.list
 
-Required  
+Required
 Type: `Array`
 
-An array containing links to diacritics sources
+An array containing links to diacritic sources.
 
 ### metadata.sources.mapping
 
-Required  
+Required
 Type: `Array`
 
-An array containing links to diacritics mapping sources
+An array containing links to diacritic mapping sources.
 
 ## data
 
-Required  
+Required
 Type: `Object`
 
-An object containing the actual mapping information. Every diacritic has its own object key. The value for each diacritic is an object specified below:
+An object containing the actual mapping information. Every diacritic has its own
+object key. The value for each diacritic is an object specified below:
 
 ### mapping
 
-TBD
+Required
+Type: `Object`
+
+An object containing mapping values for the given diacritic. This must contain a
+base or decompose value, or both. It can not be empty.
+
+### mapping.base
+
+Optional
+Type: String
+
+This is the base of the diacritic character (e.g. `�` has a base of `u`, an
+unaccented character).
+
+### mapping.decompose
+
+Optional
+Type: String
+
+This is the character, or combination of characters used to represent the
+diacritic character (e.g. `�` decomposes into `ue` in German).
+
+### equivalents
+
+Optional
+Type: `Array`
+
+Equivalents are contained within the [[equivalents file]] and are concatenated
```

Comment:
Oh, oops... that's how you link to other files in the same folder when using the GitHub wiki.